posting/views.py

```diff
@@ -60,9 +60,9 @@ def get(self, request):
                 "like_status"               : True if posting.postinglike_set.filter(user=user) else False,
                 "scrap_status"              : True if posting.postingscrap_set.filter(user=user) else False,
```

Comment:
```suggestion
                "like_status"               : posting.postinglike_set.filter(user=user).exists()
                "scrap_status"              : posting.postingscrap_set.filter(user=user).exists(),
```
파이썬에서는 if 조건문 평가시, 반환 되는 값 자체가 boolean 일 경우나, Truthy, Falsy 한 값일 경우,  == False == None == 0등으로 평가하지 않고, 그 값을 그대로 사용합니다. 그 자체로도 True or False를 판단할 수 있기 때문입니다. False한 지 평가하실 때는 not 을 활용해보세요 :)

Reply:
승현님이 리뷰주신 내용도 이에 대한 코멘트입니다 :)

Reply:
나중에 수정해보세용!

Reply:
오.... 저렇게하는게 더 좋겠네요ㅠㅠ 감사합니다!!!!! 꼭 수정해보겠습니당!!